contrib/runners/orquesta_runner/orquesta_runner/orquesta_runner.py

```diff
@@ -82,7 +83,22 @@ def _construct_st2_context(self):
 
         return st2_ctx
 
+    def _get_rerun_options(self):
+        return self.context.get('re-run', {})
+
     def run(self, action_parameters):
+        rerun_options = self._get_rerun_options()
+        tasks = rerun_options.get('tasks', None)
+        rerun = self.rerun_ex_ref and tasks
+
+        if rerun:
+            result = self.rerun_workflow(ex_ref=self.rerun_ex_ref, rerun_options=tasks)
```

Comment:
Why not pass the entire rerun options?  There's the `reset` flag in the rerun options that you will need to pass.  

```
result = self.rerun_workflow(ex_ref=self.rerun_ex_ref, options=rerun_options)
```

Reply:
From my understanding, `reset` default value is set to `True`, it means a new `ActionExecution` is created during rerun task.  If we support `reset=False`, current design is not going to work since new `ActionExecution` always created. This is why only task names are passed.

Reply:
Well, can you tell me how Mistral handles with items task?